app/controllers/api_resources_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ApiResourcesController < ApiApplicationController
+  def show
+    resource = Resource.find(params[:resource_id])
+    render json: {
+      'resource': resource.attributes.slice('id', 'name', 'url'),
+      'average_evaluation': resource.average_evaluation.to_f
```

Comment:
It would be better to round the average evaluation to 2 decimals (.round(2)), so it's displays shorter.

Reply:
I think I disagree. 
That's thinking in a particular use case, that is for displaying purposes. Then, I think the right thing to do is to round it up in the front, but it doesn't seem good to me to retrieve a value that it's not exactly what it says it is.

Just as an example, if you want to calculate an average of all resources evaluations with this information, you wouldn't have the correct information. The endpoint should be as free of business logic as possible.

Reply:
I agree with @rtichauerv. Is the front the responsible of displaying the data in a particular way. The back just passes the data without any formatting